datasets/cl/info_probidad/crawler.py

```diff
@@ -0,0 +1,48 @@
+from pantomime.types import CSV
+from urllib.parse import urlparse
+from typing import Dict
+import os
+import csv
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    person = context.make("Person")
+
+    id = os.path.basename(urlparse(row.pop("UriDeclarante")).path)
+    person.id = context.make_slug(id)
+
+    h.apply_name(
+        person,
+        first_name=row.pop("Nombre"),
+        patronymic=row.pop("ApPaterno"),
+        matronymic=row.pop("ApMaterno"),
+    )
+
+    position_name = row.pop("Cargo")
+    position = h.make_position(
+        context, position_name, country="cl", lang="spa"
+    )
+    position_lookup = context.lookup("positions", position_name)
+    
+    # all positions (ie. Cargo) should be explicitly classified as either pep or no-pep
+    if not position_lookup:
+        context.audit_data(position.to_dict())
+    
+    categorisation = categorise(context, position, is_pep = getattr(position_lookup, "is_pep", None))
```

Comment:
let's do the lookup on Cargo only, but then make position name the concatenated position, so that `Governor, Region x` and `Governor, Region y` are distinct positions

Reply:
Got it! Done in https://github.com/opensanctions/opensanctions/pull/512/commits/34a947ed64f58418a79b89a1db2e859e5557d3d4